test-plans/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name = "testcases"
+version = "0.1.0"
+
+# Required to avoid conflicts with parent workspace.
+[workspace]
+
+[dependencies]
+anyhow = "1"
+async-trait = "0.1.58"
+env_logger = "0.9.0"
+futures = "0.3.1"
+if-addrs = "0.7.0"
+log = "0.4"
+redis = { version = "0.22.1", features = ["tokio-native-tls-comp", "tokio-comp"] }
+tokio = { version = "1.24.1", features = ["full"] }
+
+libp2p = { git = "https://github.com/libp2p/rust-libp2p", branch = "master", default_features = false, features = ["websocket", "quic",  "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros", "webrtc"] }
+testplan = { version = "0.2.0", git = "https://github.com/jxs/test-plans/", branch = "add-webrtc-transport-multidim-interop" }
```

Comment:
I'd rather not have this dependency in here. Can we just copy the code over?

Reply:
Agree with copying the code. I'll add that in the longer term, all the ping implementations are in the implementation repo. The `test-plans` repo is simply a source of truth of what versions are supported and what they support. But the building/code for the impl should be in the implementation repo.

Reply:
> Agree with copying the code. I'll add that in the longer term, all the ping implementations are in the implementation repo. The `test-plans` repo is simply a source of truth of what versions are supported and what they support. But the building/code for the impl should be in the implementation repo.

What he said 😄